types/wlr_output.c

```diff
@@ -560,6 +560,17 @@ void wlr_output_schedule_frame(struct wlr_output *output) {
 		wl_event_loop_add_idle(ev, schedule_frame_handle_idle_timer, output);
 }
 
+void wlr_output_send_present(struct wlr_output *output, struct timespec *when,
+		unsigned seq, uint32_t flags) {
+	struct wlr_output_event_present event = {
+		.output = output,
```

Comment:
What's the reason for adding the output pointer here?
I'd expect it to be derivable from the actual handler that's called.

Reply:
wlroots has no idea on which output the surface is on. How could you "derive it from the handler"?

Reply:
Oh, yeah, sorry, I misunderstood. I just added it for consistency with other events. It's not strictly necessary I guess.